src/libcrun/utils.c

```diff
@@ -386,6 +386,54 @@ check_running_in_user_namespace (libcrun_error_t *err)
   return strstr (buffer, "4294967295") ? 0 : 1;
 }
 
+static int selinux_enabled = -1;
+static int apparmor_enabled = -1;
+
+int
+libcrun_initialize_selinux (libcrun_error_t *err)
+{
+  cleanup_free char *out = NULL;
+  cleanup_close int fd;
+  size_t len;
+  int ret;
+
+  if (selinux_enabled >= 0)
+    return selinux_enabled;
+
+  fd = open ("/proc/mounts", O_RDONLY | O_CLOEXEC);
```

Comment:
I am worried about this working correctly within a container, where we are setting SELinux to disabled for the contents of the container.
is_selinux_enabled() works correctly for this.  Why are you removing is_selinux_enabled?

Reply:
Is this because the user could have faked out the /proc?

Reply:
this is done on the host before we setup the new pivot_root, so we are assuming the `/proc` is correct.  I've changed it only to behave similarly to what we are doing with AppArmor, and possibly to drop the dependency in future on libselinux/libarmor since we are not using them anymore.